### PR TITLE
Chore(cmd): Add pause util to start a process in paused state and actions to perform release and upload artifact

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: uditgaurav <udit@chaosnative.com>
**What this PR does / why we need it**:

- This PR adds `pause` util which can be used to spin up a process in paused state.
- Add multiarch binaries for nsenter, pause.
- Add actions to do release and upload release artifact.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ispeakc0de — APPROVED** [review]

**Comment by uditgaurav** [comment]
- Will add the binaries for both archs

**Comment by uditgaurav** [comment]
It is ready for review @ksatchit @ispeakc0de !

**Comment by uditgaurav** [comment]
Tested run - https://github.com/uditgaurav/logs/releases/tag/2.5

**Review by ispeakc0de — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    name: create_release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: prepare binaries for release artifact
+        run: |
+          mkdir -p binaries
+          declare -a ARCH=("amd64" "arm64")
+          for val in ${ARCH[@]}; do
+            mv custom/nsutil/nsutil-linux-${val} binaries/nsutil-linux-${val}
```
Comment (ksatchit) [inline .github/workflows/release.yml:29]: We can identify an approach that can push binaries into release bundle but not maintain it locally. I guess we could take this improvement in another PR.

**Review by ksatchit — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (uditgaurav) [reply .github/workflows/release.yml]: Have made this change @ksatchit !

Now we are only keeping promeql cli binary in the repo and removed rest.

**Review by ksatchit — APPROVED** [review]

File: custom/pause/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/litmuschaos/test-tools/custom/pause
```
Comment (ispeakc0de) [inline custom/pause/go.mod:1]: do we need this file?

Comment (uditgaurav) [reply custom/pause/go.mod:1]: We can remove it also

**Review by ksatchit — APPROVED** [review]

**Pull request merged**